pandas/tests/test_internals.py

```diff
@@ -286,6 +288,36 @@ def test_repr(self):
         pass
 
 
+class TestDatetimeBlock(tm.TestCase):
```

Comment:
these are fine, but need tests for the actual issues (you can put in ``test_series`` or ``test_frame`` (find an appropriate place)